ambry-clustermap/src/test/java/com.github.ambry.clustermap/ClusterMapManagerTest.java

```diff
@@ -137,7 +141,7 @@ public void addNewPartition()
   }
 
   @Test
-  public void bestEffortAllocation()
+  public void bestEffortAllocationNonRackAware()
```

Comment:
Is it because the variable does not matter when `disableRackAwareness` is `true`?